hw/formal/tools/jaspergold/jaspergold.hjson

```diff
@@ -0,0 +1,10 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd: "{job_prefix} /edascratch/ms_edatools/cadence/jasper/2020.03/bin/jg"
```

Comment:
This looks like an internal Google path. Do we check them in normally? I would have thought it would be better to ensure that `jg` was on `$PATH`, which would mean people outside of Google could use the tool.

Reply:
Thank you for the good catch, i did not mean to add this! Thanks again. Will update that